src/collections/_documentation/data-management/advanced-datascrubbing.md

```diff
@@ -76,7 +76,22 @@ Sentry does not know if a local variable that looks like a credit card number ac
 
 ## Selectors
 
-Selectors allow you to restrict rules to certain parts of the event. This is useful to unconditionally remove certain data by variable/field name from the event, but can also be used to conservatively test rules on real data.
+Selectors allow you to restrict rules to certain parts of the event. This is useful to unconditionally remove certain data by variable/field name from the event, but can also be used to conservatively test rules on real data. A few examples:
```

Comment:
```suggestion
Selectors allow you to restrict rules to certain parts of the event. This is useful to unconditionally remove certain data by attribute name from the event, but can also be used to conservatively test rules on real data. A few examples:
```

I see we use "attribute" a lot of times in the docs. I'm good with either "field" or "attribute", however I imagine that "variable" is not very precise. WDYT?

Reply:
Let's do that